rm/rm-server/build.gradle

```diff
@@ -21,8 +21,8 @@ dependencies {
     testCompile "org.objectweb.proactive:programming-extension-pnpssl:${programmingVersion}"
     testCompile project(':common:common-api').sourceSets.test.output
 
-    testCompile 'org.jruby:jruby-complete:9.0.5.0'
-    testCompile 'org.python:jython-standalone:2.7.0'
+    testCompile 'org.jruby:jruby-complete:9.1.12.0'
+    testCompile 'org.python:jython-standalone:2.7.1'
```

Comment:
The jython version 2.7.1. seems to refer to the 2.7.1b3 which is beta3.
I think using a beta version is too risky, because jython is extensively used.